decentraland/unity-renderer
# feat: add preview mode menu

+adds debug menu button for `preview mode`.

how to test:
1- open https://play.decentraland.zone/?renderer-branch=feat/add-preview-menu&DEBUG_MODE
2- paste the following in your browser's console `globalThis.clientDebug.unityInterface.SendMessageToUnity('Main', 'TogglePreviewMenu', JSON.stringify({ enabled: true }))`

(menu button should appear right next to the minimap)

note: bounding box toggle will not work since it's not enable by default, if you want to also test that you can paste the following in the console
```
for(let sceneId of globalThis.sceneWorkers.keys()){
    if (sceneId !== "dcl-gs-avatars")
        clientDebug.ToggleSceneBoundingBoxes(sceneId,true)
}
```
